application/build.xml

```diff
@@ -363,7 +363,6 @@
 		</zip>
 		<taskdef name="py-run" onerror="report" classname="org.pyant.tasks.PythonRunTask" classpath="${dir.tools}/pyAntTasks-1.3.jar"/>
 		<py-run script="${dir.postjad.new}/utilities/submit_build.py"/>
-		<delete dir="${build.zip.dir}"/>

```

Comment:
no issues I can tell

On Wed, Feb 4, 2015 at 12:25 PM, ctsims <notifications@github.com> wrote:

> In application/build.xml
> <https://github.com/dimagi/commcare/pull/47#discussion_r24100940>:
>
> > @@ -363,7 +363,6 @@
> >  		</zip>
> >  		<taskdef name="py-run" onerror="report" classname="org.pyant.tasks.PythonRunTask" classpath="${dir.tools}/pyAntTasks-1.3.jar"/>
> >  		<py-run script="${dir.postjad.new}/utilities/submit_build.py"/>
> > -		<delete dir="${build.zip.dir}"/>
>
> Do we need to move this to before the so that workspaces which already
> have the file around won't be overwritten? Did you try to run it twice in a
> row locally and make sure there weren't any issues?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare/pull/47/files#r24100940>.
>